src/icons/HomeButton.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
```

Comment:
I would maybe change the filename, HomeButton implies a button, in this case you want to export an icon. I would maybe use "House" or whatever is actually shown on the Icon :)

Reply:
Changed it, thanks!